defaults/main.yml

```diff
@@ -112,7 +118,7 @@ patroni_bootstrap_initdb:
   - { option: "encoding", value: "UTF8" }
   - { option: "data-checksums" }
 
-patroni_bootstrap_pg_hba: []
+#patroni_bootstrap_pg_hba:

```

Comment:
This variable is extremely important and we cannot just remove it. Patroni provides two ways to configure hba rules: in [bootstrap](https://patroni.readthedocs.io/en/latest/SETTINGS.html#bootstrap-configuration) or in [postgresql](https://patroni.readthedocs.io/en/latest/SETTINGS.html#postgresql) sections. And we have to support both methods as well.